libs/utils/decorate.js

```diff
@@ -96,3 +96,22 @@ export function decorateTextOverrides(el, options = ['-heading', '-body', '-deta
     el.classList.remove(override);
   });
 }
+
+export function decorateIconStack(el) {
```

Comment:
Since this is only used in aside.js, leave in that file.  If we find a need to use it in multiple files then can move it here.

Reply:
@chrischrischris The same decorate function is also required for icon stack enhancement requested in Media block and hence it has been added in the decorate file.